README.md

```diff
@@ -237,28 +237,25 @@ We have automatic builds set up, so commits to master will trigger a build of ht
 
 ### Development usage
 
-If you want to use Docker in development, then you can build it locally with:
+If you want to use Docker in development, you can build and run the image locally with either docker-compose:
 
-Using docker-compose
 ```bash
 git clone https://github.com/solid/node-solid-server
 cd node-solid-server
 docker-compose up -d
 ```
 
-or build image manually and run it:
+or these manual commands:
```

Comment:
M-dash leading into continued text is correct.  Closing colon is also not correct; M-dash is again.
```suggestion
— or these manual commands —
```